tools/static-assets/skel-full/imports/api/links/methods.js

```diff
@@ -1,19 +1,24 @@
 // Methods related to links
 
 import { Meteor } from 'meteor/meteor';
-import { Links } from './links.js';
 import { check } from 'meteor/check';
+import { Links } from './links.js';
 
 Meteor.methods({
   'links.insert'(title, url) {
     check(url, String);
     check(title, String);
 
     // Check if this is a valid url
-    if (!url.match(/((http|https)\:\/\/)+[a-zA-Z0-9\.\/\?\:@\-_=#]+\.([a-zA-Z0-9\&\.\/\?\:@\-_=#])*/g)){
-      throw new Meteor.Error('Bad url. I.e https://www.meteor.com');
+    const re = /((http|https)\:\/\/)+[a-zA-Z0-9\.\/\?\:@\-_=#]+\.([a-zA-Z0-9\&\.\/\?\:@\-_=#])*/g;
```

Comment:
In general, I dislike having such a subjective RegEx in this sample and I'd almost propose changing the entire method to something else (e.g. `colors.insert` *shrug*) based purely for that reason.

As to the RegEx itself, it doesn't need the `/g` modifier and lacks any anchors (e.g. `^`, `$`).  Honestly, I hope this isn't used to actually verify a URL anywhere.  For example, it would match:

    hey sup regex http://http://http://?. <script><!-- // (c) 1997 //--></script>

...as a valid URL, amongst many other things.

I propose we get rid of this in any way possible.